lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -2,42 +2,58 @@ module ActiveModel
   class Serializer
     class Adapter
       class Attributes < Adapter
+        attr_accessor :_nested_associations
```

Comment:
Exactly. I've actually been looking at the ember data adapters, and it looks like everything inherits from the RESTAdapter, and only modifies what they need to, like key format, and URL format, like camelcase vs under score